packages/oui-responsive/_mixins.less

```diff
@@ -9,8 +9,7 @@
 }
 
 .desktop(@rules) {
-  @minWidth: (@oui-responsive-breakpoint-tablet + 1px);
-  @media screen and (min-width: @minWidth) {
+  @media screen and (min-width: @oui-responsive-breakpoint-tablet) {
```

Comment:
It's strange to fix it in this pr, I approve but it should be in another pull-request

Reply:
Don't sure about this change, the additional pixel had been set to prevent tablets in landscape format to display a desktop layout...

Reply:
I'll remove this change. It will be done with https://github.com/ovh-ux/ovh-ui-kit/pull/142.
The additional pixel was remove because there was a gap with:
- max-width: (@oui-responsive-breakpoint-tablet - 1px)
- min-width: (@oui-responsive-breakpoint-tablet + 1px)
To get only `<1024px` and `>=1024px`